### PR TITLE
docs: adjust readme for git system

### PR DESCRIPTION
**Pull request by sparsick**
The release instruction related to a SVN system. I adjust it so that it match the GIT system.

### CODE REVIEW

File: maven/README.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
 push-gwt.sh packages and deploys GWT to a local or remote Maven
 repository
  
-To build GWT from trunk and deploy to your local repo:
+To build GWT from main and deploy to your local repo:
 
-> cd trunk
+> git checkout main
 > ant clean dist-dev
 > maven/push-gwt.sh
```
Comment (niloc132) [inline maven/README.txt:8]: One more request (and this is also my fault for changing it), can you adjust the push-gwt script name to be `maven/push-gwtproject.sh`? The old push-gwt will only push com.google.gwt artifacts, which we don't expect to release again. I failed to update this correctly when I made that change for the last release.

File: maven/README.txt [hunk 2]
```diff
@@ -1,9 +1,9 @@
 push-gwt.sh packages and deploys GWT to a local or remote Maven
 repository
  
-To build GWT from trunk and deploy to your local repo:
+To build GWT from main and deploy to your local repo:
 
-> cd trunk
+> git checkout main
```
Comment (tbroyer) [inline maven/README.txt:6]: nit: `git switch main`

```suggestion
> git switch main
```

File: maven/README.txt (hunk 1 above)
Comment (sparsick) [reply maven/README.txt]: no problem

File: maven/README.txt (hunk 2 above)
Comment (sparsick) [reply maven/README.txt]: Thanks for the hint. I adjusted it.

**Review by niloc132 — APPROVED** [review]

**Pull request merged**